src/jdk.internal.le/share/classes/jdk/internal/org/jline/terminal/impl/ffm/Kernel32.java

```diff
@@ -0,0 +1,927 @@
+/*
+ * Copyright (c) 2009-2023, the original author(s).
+ *
+ * This software is distributable under the BSD license. See the terms of the
+ * BSD license in the documentation provided with this software.
+ *
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+package jdk.internal.org.jline.terminal.impl.ffm;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.VarHandle;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+@SuppressWarnings({"unused", "restricted"})
+final class Kernel32 {
+
+    public static final int FORMAT_MESSAGE_FROM_SYSTEM = 0x00001000;
```

Comment:
Here is another bunch of constants that seem generated. Is this file created by jextract? (I'm sorry, I have not played around enough with jextract to recognize the output it generates).

In any way, some documentation on how this file was created, if any kind of automation was involved, would be nice.